index.html

```diff
@@ -1,71 +1,268 @@
+                    <a href="#"><img src="images/microverse.svg" alt="M icon" /></a>
+                  </li>
+                  <li>
+                    <a href="#"
+                      ><img src="images/github.svg" alt="Github icon"
+                    /></a>
+                  </li>
+                  <li>
+                    <a href="#"
+                      ><img src="images/twitter.svg" alt="Peace icon"
+                    /></a>
+                  </li>
+                </ul>
+                <div class="project-details-btn">
+                    <input type="button" value="Get my resume" />
+                  </div>
+        </div>
+        <div class="bottom">
+            <ul class="box">
+                <li>
+                    <div class="skills">
+                        <p><strong>Languages</strong></p>
+                        <i class="fa fa-chevron-down" aria-hidden="true"></i>
+                    </div>
+                </li>
+                <li >
+                    <ul class="skills-languages">
+                        <li>
+                            <img src="images/js.svg" alt="">
+                            <p>JavaScript</p>
+                        </li>
+                        <li>
+                            <img src="images/html.svg" alt="">
+                            <p>HTML</p>
+                        </li>
+                        <li>
+                            <img src="images/css.svg" alt="">
+                            <p>CSS</p>
+                        </li>
+                    </ul>
+                </li>
+                <li class="skills">
+                    <p><Strong>Frameworks</Strong></p>
+                    <i class="fa fa-chevron-right"></i>
+                </li>
+                <hr/>
+                <li class="skills">
+                    <p><Strong>Skills</Strong></p>
+                    <i class="fa fa-chevron-right"></i>
+                </li>
+                <hr/>
+            </ul>
+        </div>
+      </section>
+      <!--End of about section-->
+
+      <!--Start of Contact Page-->
+      <section class="contact-page">
+        <div class="contact-block">
+            <div class="description">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!</p>
+            </div>
+            <form action="#">
+                <fieldset>
+                    <input type="text" placeholder="Lorem Ipsum">
+                    <input type="email" placeholder="loremipsum@gmail.com">
+                    <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here..."></textarea>
+                </fieldset>
+                <button type="submit">Get in touch</button>
+            </form>
+            <div class="blackline"></div>
+        </div>
+      </section>
```

Comment:
- [Optional] Kindly consider removing the contact section and stash it somewhere. The contact section will be added in an upcoming milestone. You are expected to build the portfolio one milestone at a time.